Remove duplicate gradle dependencies

lhoyong opened the pull request:
`libs.android.tools.build.gradle` is same for `libs.android.gradlePlugin`

lhoyong commented:
@Goooler  I resolved conflict.

bentrengrove commented:
This conflict resolution doesn't look right to me? We just changed that variable name to `lintMinCompose` and this would revert it

lhoyong commented:
> This conflict resolution doesn't look right to me? We just changed that variable name to `lintMinCompose` and this would revert it

@bentrengrove oh thanks. i fixed it
